collector.go

```diff
@@ -285,6 +294,41 @@ func queryNamespaceMappings(ch chan<- prometheus.Metric, db *sql.DB, metricMap m
 	return namespaceErrors
 }
 
+// Gather the pgbouncer version info.
+func queryVersion(ch chan<- prometheus.Metric, db *sql.DB) error {
+	rows, err := db.Query("SHOW VERSION;")
+	if err != nil {
+		return errors.New(fmt.Sprintf("error getting pgbouncer version: %v",  err))
+	}
+	defer rows.Close()
+
+	var columnNames []string
+	columnNames, err = rows.Columns()
+	if err != nil {
+		return errors.New(fmt.Sprintf("error retrieving column list for version: %v", err))
+	}
+	if len(columnNames) != 1 || columnNames[0] != "version" {
+		return errors.New("show version didn't return version column")
+	}
+
+	var bouncerVersion string
+
+	for rows.Next() {
+		err := rows.Scan(&bouncerVersion)
+		if err != nil {
+			return err
+		}
+		ch <- prometheus.MustNewConstMetric(
+			bouncerVersionDesc,
+			prometheus.GaugeValue,
+			1.0,
+			bouncerVersion,
```

Comment:
This ends up returning the full query return value which is a bit unnecessary.  ` PgBouncer 1.12.0`  Any way we can shrink this down to _only_  `1.12.0`?

Reply:
We could, but I would rather keep it simple and provide the verbatim string that is returned by the bouncer.